extensions/security/runtime/src/main/java/io/quarkus/security/runtime/interceptor/SecurityHandler.java

```diff
@@ -16,9 +19,6 @@
 @Singleton
 public class SecurityHandler {
 
-    private static final String HANDLER_NAME = SecurityHandler.class.getName();
-    private static final String EXECUTED = "executed";
```

Comment:
Shouldn't these be replaced by the constants in `SecurityHandlerConstants`?

Reply:
Sorry, I can't comprehend - SecurityHandler constants have been moved to SecurityHandlerConstants as we to access them from RR (would come with alternative mechanism like 'anything with xy there...', but this looks more reliable). RR set these constants to prevent the check. Can you explain in more detail how you mean it?

Reply:
Aren't these same constants defined in `io.quarkus.security.spi.runtime.SecurityHandlerConstants`?

Reply:
Exactly, that's why they are already removed from SecurityHandler (it has red background). I'll need a little more, sorry.

Reply:
Nevermind, reviewing from the IDE tricked me...

Reply:
np